### ISSUE TITLE
Trigger an automation using IFTTT

### ISSUE THREAD

**Issue by lyance** [ilcato/homebridge-ifttt#14]
Hi there, is it possible to use IFTTT to trigger an automation to be executed?  I've got devices that support IFTTT but are not supported currently by HomeKit/homebridge. It would be nice to have. I understand this plugin does the opposite. Thanks!

**Comment by ilcato** [comment]
@lyance, describe your use case please.
With this plugin you can generate an event FROM HomeKit TO IFTTT.

**Comment by lyance** [comment]
Thanks for replying @ilcato. One use case I have in mind is using a IFTTT-enabled camera that acts as a motion sensor. If I would like a HomeKit scene to be activated when the camera detects motion (which is the If-part of an IFTTT applet ) - I wish there was a way to let HomeKit use the IFTTT-enabled camera as a sensor.

**Comment by ilcato** [comment]
not sure it is physically possible.

**Comment by lprhodes** [comment]
This is definitely possible...the plugin would need to be able to toggle the virtual device state based on an ifttt request. A HomeKit automated would then have a trigger based on the the virtual devices state changing.

**Comment by iRonin** [comment]
Try more advaced app than Apple's Home app.

**Comment by hepcat72** [comment]
I have a different use-case that perhaps is somewhat similar.  I have devices that I've set homebridge up to control using homebridge-commander.  Those devices cannot be controlled via IFTTT or homekit directly (they are Etekcity outlets).  What I'm looking for is the ability to setup IFTTT applets that can turn those outlets on/off (and additionally/optionally have their state updated in the Home app as well).

So is it possible to have an IFTTT trigger to flip a home app switch/accessory that has been configured for a different homebridge plugin?

**Comment by githubberboy** [comment]
If I understand what you’re looking for, I think that you have to use webhooks applet in IFTTT (if your devices accept http post/get commands, of course).

**Comment by hepcat72** [comment]
No, my Etekcity devices are cheap RF outlets and can only be controlled via a 433Mhz signal. I have a remote that came with the outlets and I also bought 433Mhz transmitter and receiver chips that are plugged into a breadboard connected to the pi.  I can turn them off & on using pilight command-line calls, which I have setup using homebridge-commander.  That means I can control them via Siri, the home app, or via script/cron-job.  But I would like to be able to turn them on/off using a variety of IFTTT triggers.  So I need a way for IFTTT to trigger on/off events of any of the devices homebridge knows about.
